src/containers/AssignmentTexterContact.jsx

```diff
@@ -971,6 +979,7 @@ const mapMutationsToProps = () => ({
             text
             isFromContact
           }
+          textingTurnedOff
```

Comment:
We might want to add this to TexterTodoList.jsx as well, to move people off the /todos screen which would be consuming DB resources.